### PR TITLE
New Chaplain Items Attempt #3

### PR DESCRIPTION
**Pull request by Ezuo**
>Cherry picked commits, only picking out the chaplain changes (No more devmeme PRs)
>Fixed up the map file, no more insane addition/deletion numbers

A box is added to the chaplains office at round start that allowed them to select from the rod, a staff, or an athame. Purely a cosmetic choice.

### CODE REVIEW

File: code/game/objects/items/weapons/weaponry.dm [hunk 1]
```diff
@@ -11,11 +11,58 @@
...
+	icon_state = "box"
+	item_state = "box"
+
+/obj/item/weapon/nullrod/itembox/attack_self(mob/user as mob)
+	if(..()) return
+
+	var/selection = input("Pick a null item type.") in list ("Rod","Staff", /*"Orb",*/ "Athame")
+	switch(selection)
+		if ("Rod")
+			new /obj/item/weapon/nullrod(usr.loc)
```
Comment (skull132) [inline code/game/objects/items/weapons/weaponry.dm]: Don't use `usr` in procs that are not directly exposed to the user. In this case, `user.loc` is better.

**Comment by Lohikar** [comment]
> `icons/obj/icons - Shortcut.lnk`

File: code/game/objects/items/weapons/weaponry.dm [hunk 2]
```diff
@@ -11,11 +11,58 @@
...
+	name = "null item box"
+	desc = "A box to safe keep your religious items. What item did you bring to work today?"
+	icon = 'icons/obj/storage.dmi'
+	icon_state = "box"
+	item_state = "box"
+
+/obj/item/weapon/nullrod/itembox/attack_self(mob/user as mob)
+	if(..()) return
+
+	var/selection = input("Pick a null item type.") in list ("Rod","Staff", /*"Orb",*/ "Athame")
```
Comment (skull132) [inline code/game/objects/items/weapons/weaponry.dm]: `in list("A", "B", "C")`. Minor style thing.

Also, why's the orb commented out?

Comment (Ezuo) [reply code/game/objects/items/weapons/weaponry.dm]: Oh, that was planned, but doesn't have a sprite yet. As soon as I get a sprite ready for it I can add it to the list.

**Pull request merged**